### ISSUE TITLE
Command to support importing an ovpn config file

### ISSUE THREAD

**Issue by jhalterman** [benwebber/tunnelblickctl#3]
Would be awesome if there were support for important an ovpn config file. My use case is that I hop on and off new VPNs throughout the day (generated for cloud infra), so I'm constantly having to import these.

**Comment by benwebber** [comment]
Great idea. I've pushed a new `install` command.

It's a little hacky: the output for `tunnelblickctl install` is not very helpful at the moment. Tunnelblick prompts you for your password, which blocks `tunnelblickctl`. However, if you enter the wrong password, it unblocks the AppleScript. So the newly installed tunnel may or may not appear in the output.

I'm opening a new issue to add a symmetric `uninstall` command. This will require a bit more work. To do it properly and remove the shadow configurations as well, we need to script the Tunnelblick GUI.

**EDIT:** I'll release a new version and binary after figuring out `uninstall`.

**Comment by benwebber** [comment]
I released `install` without `uninstall` in 0.2.0.
